v2/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/nats-io/jwt/v2
+
+require (
+	github.com/nats-io/jwt v0.3.2
```

Comment:
I think we should consider tagging v1.0.0 first before merging in the v2 commits in master

Reply:
this is probably ok.

Reply:
note that we are not releasing v2 until later - the merge won't trigger a v2 release until the other systems consume them.

Reply:
Imho I would imagine that jetstream is going to be the feature that triggers a nats-server version bump (or at least hoping that is the case). I think that is the opportunity to v2 releases. This will draw the line that a previous major version can reject the older jets. Otherwise, it seems that we are not applying semver properly.

Reply:
It might be fine but if we don't know we probably shouldn't do it unless testing that is ok, in general should be careful since there isn't really room for errors with Go modules because the Go proxy will cache the state of the repo as soon as it is available.